### PR TITLE
feat(subscribe): L4-3126 changed styles

### PR DESCRIPTION
**Pull request by constantinehuzenko**
**Jira ticket**

[L3-3126](https://phillipsauctions.atlassian.net/browse/L3-3126)

**Screenshots**
| Before | After |
| ------ | ----- |
| image | image |

**Summary**

Changed styles to follow new Figma's design

**Change List (describe the changes made to the files)**

- feat(_subscribe.scss): added new styles
- feat(Subscribe.tsx): added privacy text

**Acceptance Test (how to verify the PR)**

- verify that subscribe states are working
- verify that privacy text is visible
- verify that signup button is full width in mobile view

**Regression Test**

- (Optional) Add verification steps to make sure this PR doesn't break old functionality

**Evidence of testing**

- Post logs, screenshots, etc

**Things to look for during review**

- [ ] PR title should correctly describe the most significant type of commit. I.e. `feat(scope): ...` if a `minor` release should be triggered.
- [ ] All commit messages follow convention and are appropriate for the changes
- [ ] All references to `phillips` class prefix are using the prefix variable
- [ ] All major areas have a `data-testid` attribute.
- [ ] Document all props with jsdoc comments
- [ ] All strings should be translatable.
- [ ] Unit tests should be written and should have a coverage of 90% or higher in all areas.

New Components

- [ ] Are there any [accessibility considerations](https://www.w3.org/WAI/ARIA/apg/patterns/) that need to be taken into account and tested?
- [ ] Default story called "Playground" should be created for all new components
- [ ] Create a jsdoc comment that has an Overview section and a link to the Figma design for the component
- [ ] Export the component and its typescript type from the `index.ts` file
- [ ] Import the component scss file into the `styles.scss` file.

[L3-3126]: https://phillipsauctions.atlassian.net/browse/L3-3126?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Subscribe/_subscribe.scss [hunk 1]
```diff
@@ -4,34 +4,57 @@
...
+    &__label {
+      color: $pure-black;
+      font-size: 0.75rem;
+      text-transform: none;
+    }
+
+    &__validation {
+      padding-left: 0;
+    }
+  }
```
Comment (davidicus) [inline src/components/Subscribe/_subscribe.scss:20]: This should probably be applied to the input component itself. I see that the padding got broken on that input at some point,

File: src/components/Subscribe/_subscribe.scss [hunk 2]
```diff
@@ -4,34 +4,57 @@
...
 .#{$px}-subscribe {
-  max-width: 30rem;
+  max-width: 600px;
   width: 100%;
 
+  .#{$px}-input {
+    &__label {
+      color: $pure-black;
+      font-size: 0.75rem;
+      text-transform: none;
```
Comment (davidicus) [inline src/components/Subscribe/_subscribe.scss:14]: This should probably be applied to the input component. From that [figma](https://www.figma.com/design/hMu9IWH5N3KamJy8tLFdyV/EASEL-Compendium%3A-Tokens%2C-Components-%26-Patterns?node-id=7584-17668&m=dev) it looks like this applies to all inputs.

Comment (davidicus) [reply src/components/Subscribe/_subscribe.scss:14]: Should the font size come from a text mixin?

File: src/components/Subscribe/Subscribe.tsx [hunk 3]
```diff
@@ -51,6 +51,10 @@ export interface SubscribeProps extends React.HTMLAttributes<HTMLFormElement> {
    * Subscribe success text
    */
   successText?: string;
+  /**
+   * Subscribe privacy text
+   */
+  privacyText?: string;
```
Comment (davidicus) [inline src/components/Subscribe/Subscribe.tsx:57]: are we going to have this privacy text in all areas where we use this component? We are making it conditional in the types but we are always rendering a p tag. We should conditionally render this element.

File: src/components/Subscribe/Subscribe.tsx
Comment (davidicus) [inline src/components/Subscribe/Subscribe.tsx:1]: while we are here we should change the jumping that happens when we have error text by rendering a `&nbsp; if no error or warning is present.

**Review by scottdickerson — APPROVED** [review]
LGTM

File: src/components/Subscribe/_subscribe.scss [hunk 4]
```diff
@@ -4,34 +4,57 @@
...
+    color: $pure-black;
   }
 
   &__input__wrapper {
     margin-top: $spacing-md;
   }
 
   &__button {
-    margin: $spacing-md 0 $spacing-md;
+    height: 36px;
```
Comment (davidicus) [inline src/components/Subscribe/_subscribe.scss:39]: I am not a fan of setting explicit heights on things. The padding, font size, and line-height should equal the height of the button. Adding up the values they have in [this figma](https://www.figma.com/design/koazePmhaPVdUszim1NGTP/Navigation?node-id=2210-1384&m=dev) does not add up to what is listed.

File: src/components/Subscribe/_subscribe.scss [hunk 5]
```diff
@@ -4,34 +4,57 @@
...
+    @include media($size-sm) {
+      width: 240px;
+    }
+  }
+
+  &__privacy {
+    @include text($body3);
+
+    // margin-top: $spacing-sm;
+    background-color: yellow;
```
Comment (davidicus) [inline src/components/Subscribe/_subscribe.scss:52]: Where in the figma are you seeing this background color applied. I can not find it.

**Review by davidicus — CHANGES_REQUESTED** [review]
A few small comments here

File: src/components/Input/_input.scss [hunk 6]
```diff
@@ -44,6 +43,7 @@ $lg: #{$px}-input--lg;
     -webkit-line-clamp: 2;
     line-height: 1.25;
     overflow: hidden;
+    padding-left: 0;
```
Comment (davidicus) [inline src/components/Input/_input.scss:46]: I think this is not needed with the removal on line 130

**Review by davidicus — COMMENTED** [review]
This all looks good. I do think the label of the input is supposed to be red  when in an error state. Currently, it is black.

**Review by davidicus — APPROVED** [review]

**Pull request merged**